config/test.exs

```diff
@@ -4,7 +4,7 @@ use Mix.Config
 # you can enable the server option below.
 config :prm, PRM.Web.Endpoint,
   http: [port: 4001],
-  server: false
+  server: true
```

Comment:
You don't need to start server when running acceptance tests, since there will be a chance that your request will hit local copy of code, not a dockerized release.